src/v/storage/tests/index_state_test.cc

```diff
@@ -17,6 +20,21 @@
 
 #include <boost/test/unit_test.hpp>
 
+#ifdef NDEBUG
+inline constexpr int num_test_runs = 100;
+inline constexpr int num_samples = 10000;
+#else
+inline constexpr int num_test_runs = 1;
+inline constexpr int num_samples = 1000;
+#endif
+
+static auto make_random_compression_config() {
+    bool use_compression = random_generators::get_int<int>() % 2 == 0;
+    scoped_config cfg;
+    cfg.get("log_segment_index_compression").set_value(use_compression);
+    return cfg;
```

Comment:
is there any information about what index format is used in case we get a failure in the tests?